less/_components/header-highlight.less

```diff
@@ -0,0 +1,35 @@
+// adds a highlight bar under the heading
+h2, h3, h4, h5 {
```

Comment:
Do we really need to specify the h2, h3, h4, h5?

I would prefer a simple class that we apply to any text element.

Reply:
We have highlights that apply to some other things. I prefixed it with header element tags to make sure we don't have any collision.

Reply:
This is harder to override the css class since we will need to specify h2.[class_name] {}

If the concern is collision, we should come up with a different name for the class.

Reply:
I do not consider this a "component". I would prefer that we move this new class in https://github.com/EclipseFdn/solstice-assets/blob/master/less/_components/classes-typography.less

After reviewing this file, I notice that we already have a .heading-underline.

I am thinking that we might just need to create variables for properties for the .heading-underline.